python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -351,6 +362,23 @@ def _apply_default_remote_args(ray_remote_args: Dict[str, Any]) -> Dict[str, Any
     def get_autoscaling_actor_pools(self) -> List[AutoscalingActorPool]:
         return [self._actor_pool]
 
+    def _manage_actor_restarting_state(self, actor):
+        actor_state = actor._get_local_state()
```

Comment:
Is this expensive to call?

Reply:
This call looks like state lookup to me.

actor.py:1561 _get_local_state()
raylet.pyx:4371 get_local_actor_state()

Reply:
it should be cheap